pom.xml

```diff
@@ -1282,7 +1282,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
       </dependency>
       <dependency>
         <groupId>org.hamcrest</groupId>
-        <artifactId>hamcrest-all</artifactId>
+        <artifactId>hamcrest-core</artifactId>
```

Comment:
Can we try to get rid of core as well?

Reply:
Sure, let me try that.

Reply:
I see https://github.com/apache/ozone/pull/5923/

Reply:
Yes, we can remove `hamcrest-core`, but only because we also have `hamcrest` as a transitive dependency.